recipes/r-stitch/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+c_compiler_version:
```

Comment:
Do older compilers not work?

Reply:
gcc 13 does not work

Reply:
And if cxx is set to 12 and c_compiler is newer it fails